src/rpc/connection.rs

```diff
@@ -423,4 +423,18 @@ impl RpcContext for Connection {
 		surrealdb::iam::verify::token(DB.get().unwrap(), &mut self.session, &token.0).await?;
 		Ok(Value::None)
 	}
+
+	async fn clear(&mut self) -> Result<impl Into<Data>, RpcError> {
+		LIVE_QUERIES.write().await.retain(|_, v| *v != self.id);
```

Comment:
There is a trade-off between memory and CPU usage here. Not sure if we should choose a two-sided `HashMap` (hence more memory usage) or keep this slow `retain` solution.

Reply:
Maybe by keeping a reference of current LQs in `RpcContext` via a `HashSet<&Uuid>`?